mono/mini/exceptions-amd64.c

```diff
@@ -919,6 +949,8 @@ mono_arch_handle_altstack_exception (void *sigctx, MONO_SIG_HANDLER_INFO_TYPE *s
 GSList*
 mono_amd64_get_exception_trampolines (gboolean aot)
 {
+	mono_cross_compile_assert_not_reached ();
+
```

Comment:
This is incorrect, and probably a bunch of others as well.

Reply:
I think I understand and will make another pass.